tigerpath/majors_and_certificates/majors/HIS.json

```diff
@@ -95,285 +100,606 @@
+                        "NES 369/HIS 251: The World of the Cairo Geniza",
+                        "EAS 280/HIS 279: Nomadic Empires",
+                        "HIS 294: What Is the Scientific Revolution?",
+                        "HIS 303: Colonial Latin America to 1810",
+                        "EAS 409/HIS 309: The Warrior Culture of Japan",
+                        "HIS 314: Precolonial Africa",
+                        "HIS 324 : Early Modern China",
+                        "CLA 326/HIS 326: Topics in Ancient History - Town and Country",
+                        "CLA 327/HIS 327: Topics in Ancient History",
+                        "CLA 324/HIS 328: Classical Historians and Their Philosophies of History",
+                        "HIS 332: India Before Europe",
+                        "EAS 317/HIS 335: The Warrior Culture of Japan",
+                        "NES 437/HIS 337: The Ottoman Empire, 1300-1800",
+                        "HIS 343: The Formation of Europe in the First Millennium",
+                        "HIS 344: The Civilization of the High Middle Ages",
+                        "HIS 345: The Crusades",
+                        "HUM 320/HIS 346: Making Medieval Worlds: Methods and Materials",
+                        "HIS 367: English Constitutional History",
+                        "HIS 371: The Colonization of North America",
+                        "HIS 413: Medieval Democracy: Italian City States of the Middle Ages",
+                        "HIS 421: Venice and the Mediterranean World",
+                        "HIS 422: Hindu, Muslim, Untouchable",
+                        "HIS 423: The History of Christianity in Africa: From St. Mark to Desmond Tutu",
+                        "HIS 437: Law after Rome",
+                        "HIS 452: Magic, Matter, Medicine: Science in the Medieval World",
+                        "HIS 491: Fertile Bodies: A Cultural History of Reproduction from Antiquity to the Enlightenment",
+                        "HIS 495: Alchemy: Art and Science"
+                      ]
+                    }
+                  ]
+                },
+                { 
+                  "name":"Geographical Distribution Requirement",
+                  "max_counted":null,
+                  "min_needed":2,
+                  "double_counting_allowed":true,
+                  "explanation": "<i>Editor's Note: This does not represent a comprehensive list of courses that fulfill this requirement as courses change from semester to semester.</i>\n\n Students in the class of 2023 are required to take at least two (2) courses that are principally focused on Africa, Asia, Latin America, or the Middle East. These courses may double count with the thematic areas.",
+                  "course_list":[
+                    "HIS *"
+                  ]
+                }
+              ],
+              "year_switch": [

```

Comment:
https://history.princeton.edu/academics/undergraduate/current-concentrators#DistributionReqs

"Class of 2021 and Class of 2022 will continue with the existing requirements. Current concentrators who wish to migrate to the new requirements should consult with the Director of Undergraduate Studies."